dev-util/umockdev/Manifest

```diff
@@ -1,2 +1,3 @@
+DIST umockdev-0.10.tar.xz 757140 BLAKE2B e1aeabf716402ff9532dae31ce7f0ffc4872eb8a4f8247f9a6fdd06ea9d09603515cc3bc3704e8e17bb579faa3619efcac38985254ba6ec7654f5363a58cb4a8 SHA512 ea26080a46b36d1b5a92c3ee6f42e2fa77826eeaf4cbb378d9454ba91a091084926cbf13de4c4398aedabb15578b03eccc5fc78f8ba2faa756de65e53253b82b
```

Comment:
Please add empty line before footer part of commit message. Also don't use additional body part if it doesn't say anything that's not already in summary.